components/button/Button.stories.ts

```diff
@@ -1,12 +1,12 @@
 import type { Meta, StoryObj } from '@storybook/react';
-import { fn } from "@storybook/test";
+import { fn } from '@storybook/test';
```

Comment:
Is this from Prettier auto-formatting?

Reply:
Correct, this is from Prettier formatting.
Should I write an issue to run Prettier on the entire codebase so all files are formatted uniformly?

Reply:
Yes, after this PR we should run prettier on the whole codebase to avoid these diffs showing up in every single PR for the foreseeable future.

Reply:
Added a ticket for this. Thanks, Mike!